src/server/routes/emailTemplates.ts

```diff
@@ -0,0 +1,52 @@
+import { Request, Router } from 'express';
+import { ResponseWithRequestState } from '@/server/models/Express';
+import { render } from 'mjml-react';
+
+import { ResetPassword } from '../../email/templates/ResetPassword/ResetPassword';
+import { ResetPasswordText } from '../../email/templates/ResetPassword/ResetPasswordText';
+import { Verify } from '../../email/templates/Verify/Verify';
+import { VerifyText } from '../../email/templates/Verify/VerifyText';
+
+const router = Router();
+
+const emailTemplateTypes = ['ResetPassword', 'Verify'] as const;
+type EMAIL_TEMPLATE_TYPE = typeof emailTemplateTypes[number];
+
+type EMAIL_RENDER_RESULT = {
```

Comment:
I think we only use ALL_CAPS for string constants, rather than types. I would suggest `EmailRenderResult` and `EmailTemplateType` naming for the above types